src/backend/distributed/commands/alter_table.c

```diff
@@ -24,6 +24,8 @@
  *-------------------------------------------------------------------------
  */
 
+#warning "This is a compilation warning for testing purposes"
```

Comment:
Added this test warning to test validation phase with a valid warning

Reply:
Lets remove this and let tests pass